### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by w2vgd**
Update examples throughout the app to make it consistent

### CODE REVIEW

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 1]
```diff
@@ -39,30 +37,32 @@
...
-                    new Deadline(LocalDate.parse("01-08-2021", DateTimeFormat.INPUT_DATE_FORMAT)),
-                    new Client(new Name("Alice"), new Contact("91234567"),
-                            new Email("alice@gmail.com"), new AskingPrice("$800,000")),
-                    getTagSet("2 bedrooms", "65 square metres")),
+                new Address("Blk 784 Woodlands Crescent #01-01"),
+                new PostalCode("731784"),
+                new Deadline(LocalDate.parse("01-08-2021", DateTimeFormat.INPUT_DATE_FORMAT)),
+                new Client(new Name("Caleb"), new Contact("84459627"),
+                    new Email("caleb_goh@gmail.com"), new AskingPrice("$350,000")),
+                getTagSet("2 bedrooms", "65 square metres")),
```
Comment (candyhy) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:56]: do u think `ClientBuilder` should be used for this?

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 2]
```diff
@@ -39,30 +37,32 @@
...
                 new Address("180 Depot Rd"),
                 new PostalCode("109684"),
                 new Deadline(LocalDate.parse("10-04-2021", DateTimeFormat.INPUT_DATE_FORMAT)),
                 new Remark("Urgent!!!"),
-                new Client(new Name("Alex Yeoh"), new Contact("87438807"),
-                    new Email("alexyeoh@example.com"), new AskingPrice("$1,000,000")),
-                    getTagSet("Labrador Park MRT", "2015 World Building of the Year")),
+                new Client(new Name("Freddy"), new Contact("87438807"),
+                    new Email("freddy_123@hotmail.com"), new AskingPrice("$1,000,000")),
+                getTagSet("Labrador Park MRT", "2015 World Building of the Year")),
```
Comment (candyhy) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:65]: same for this

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 3]
```diff
@@ -39,30 +37,32 @@
...
+                new Deadline(LocalDate.parse("01-08-2021", DateTimeFormat.INPUT_DATE_FORMAT)),
+                new Client(new Name("Caleb"), new Contact("84459627"),
+                    new Email("caleb_goh@gmail.com"), new AskingPrice("$350,000")),
+                getTagSet("2 bedrooms", "65 square metres")),
             new Property(new Name("The Interlace"),
                 new Type("Condo"),
                 new Address("180 Depot Rd"),
                 new PostalCode("109684"),
                 new Deadline(LocalDate.parse("10-04-2021", DateTimeFormat.INPUT_DATE_FORMAT)),
                 new Remark("Urgent!!!"),
```
Comment (candyhy) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:62]: should `PropertyBuilder` class be used?

File: src/main/java/seedu/address/model/util/SampleDataUtil.java [hunk 4]
```diff
@@ -71,49 +71,47 @@
...
+                new Date(LocalDate.parse("12-10-2021", DateTimeFormat.INPUT_DATE_FORMAT)),
+                new Time(LocalTime.parse("1400", DateTimeFormat.INPUT_TIME_FORMAT))),
+            new Appointment(new Name("Meet Emily"),
+                new Remark("At MayFair Gardens"),
+                new Date(LocalDate.parse("15-06-2021", DateTimeFormat.INPUT_DATE_FORMAT)),
+                new Time(LocalTime.parse("1100", DateTimeFormat.INPUT_TIME_FORMAT))),
+            new Appointment(new Name("Meet Freddy"),
+                new Remark("At Orchard Centrepoint's Starbucks"),
+                new Date(LocalDate.parse("17-08-2021", DateTimeFormat.INPUT_DATE_FORMAT)),
+                new Time(LocalTime.parse("1930", DateTimeFormat.INPUT_TIME_FORMAT))),
```
Comment (candyhy) [inline src/main/java/seedu/address/model/util/SampleDataUtil.java:110]: should `AppointmentBuilder` class be used for this?

File: src/test/java/seedu/address/testutil/TypicalModelManager.java [hunk 5]
```diff
@@ -1,27 +1,27 @@
...
+
+    ///**
+    // * Returns an {@code AppointmentBook} with all the typical appointments.
+    // */
+    //public static ModelManager getTypicalModelManager() {
+    //    ModelManager modelManager =
+    //            new ModelManager(TypicalAppointments.getTypicalAppointmentBook(),
+    //                    TypicalProperties.getTypicalPropertyBook(), new UserPrefs());
+    //    return modelManager;
+    //}
```
Comment (candyhy) [inline src/test/java/seedu/address/testutil/TypicalModelManager.java]: are these supposed to be commented out?

File: src/test/java/seedu/address/testutil/TypicalProperties.java [hunk 6]
```diff
@@ -55,12 +64,13 @@ public static PropertyBook getTypicalPropertyBookWithClient() {
...
-        return new ArrayList<>(Arrays.asList(WOODLANDS_CRESCENT, MAYFAIR, BURGHLEY_DRIVE));
+        return new ArrayList<>(Arrays.asList(MAYFAIR, BURGHLEY_DRIVE, WOODLANDS_CRESCENT, JURONG));
     }
 
-    public static List<Property> getTypicalPropertiesWithClient() {
-        return new ArrayList<>(Arrays.asList(WOODLANDS_CRESCENT, MAYFAIR, BURGHLEY_DRIVE, JURONG));
-    }
+    //public static List<Property> getTypicalPropertiesWithClient() {
+    //    return new ArrayList<>(Arrays.asList(WOODLANDS_CRESCENT, MAYFAIR, BURGHLEY_DRIVE, JURONG));
+    //}
```
Comment (candyhy) [inline src/test/java/seedu/address/testutil/TypicalProperties.java]: this too

**Review by candyhy — COMMENTED** [review]
LGTM with some comments

File: src/main/java/seedu/address/model/util/SampleDataUtil.java (hunk 1 above)
Comment (w2vgd) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java:56]: I think we should probably separate source code and test code

File: src/main/java/seedu/address/model/util/SampleDataUtil.java (hunk 2 above)
Comment (w2vgd) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java:65]: Same as above

File: src/main/java/seedu/address/model/util/SampleDataUtil.java (hunk 3 above)
Comment (w2vgd) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java:62]: Same as above

File: src/main/java/seedu/address/model/util/SampleDataUtil.java (hunk 4 above)
Comment (w2vgd) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java:110]: Same as above

File: src/test/java/seedu/address/testutil/TypicalModelManager.java (hunk 5 above)
Comment (w2vgd) [reply src/test/java/seedu/address/testutil/TypicalModelManager.java]: Oops, forgot to remove

File: src/test/java/seedu/address/testutil/TypicalProperties.java (hunk 6 above)
Comment (w2vgd) [reply src/test/java/seedu/address/testutil/TypicalProperties.java]: will remove them

File: src/main/java/seedu/address/model/util/SampleDataUtil.java (hunk 1 above)
Comment (candyhy) [reply src/main/java/seedu/address/model/util/SampleDataUtil.java:56]: ok

**Review by candyhy — APPROVED** [review]
LGTM

**Pull request merged**